man/rgbasm.5

```diff
@@ -1868,9 +1870,9 @@ can optionally be given an integer parameter, and will apply the above shifting
 A negative parameter will shift the arguments in reverse.
 .Pp
 .Ic SHIFT
-is useful in
+is especially useful in
 .Ic REPT
-blocks to repeat the same commands with multiple arguments.
+loops, to repeat the same commands but with different arguments each time.
```

Comment:
Ever since we added support for `\<10>` and `\<N>`, it has become less necessary to `SHIFT` arguments. Should we mention that here?

Reply:
Perhaps briefly: "is especially useful in loops as an alternative to `\<var>`..."

Reply:
`SHIFT` and `\2` work where `\<N+1>` doesn't.

Reply:
The previous two common `SHIFT` idioms were:

```x86asm
MACRO many_args
    db \1, \4, \9
    shift
    db \9 ; tenth
ENDM
```

and:

```x86asm
MACRO each
    db \1
    SHIFT
    REPT _NARG
        dw \1
        SHIFT
    ENDR
ENDM
```

Both of these can be handled without `SHIFT` now:

```x86asm
MACRO many_args
    db \1, \4, \9, \<10>
ENDM
```

and:

```x86asm
MACRO each
    db \1
    FOR N, 2, _NARG
        dw \<N>
    ENDR
ENDM
```

There may be some more complex cases when `SHIFT` is still more convenient, but I wouldn't like the docs to recommend it as standard in these typical cases.

Reply:
But you can't do
```asm
REPT _NARG / 2
    barrel_roll \1, \2
    shift 2
ENDR
```

...so `SHIFT` still has uses inside of `REPT`s.

Reply:
Yes, as I said, there are some more complex cases when `SHIFT` is still more convenient, but I wouldn't like the docs to recommend it as standard in typical cases.

Reply:
This is not recommending SHIFT as standard for those cases, but stating that those cases are standard for SHIFT, essentially. To explain what it's useful for.

Reply:
Okay, I'll stop bikeshedding this. :P